src/oscar/apps/checkout/views.py

```diff
@@ -553,7 +553,7 @@ def submit(self, user, basket, shipping_address, shipping_method,  # noqa (too c
         # created).  We also save it in the session for multi-stage
         # checkouts (eg where we redirect to a 3rd party site and place
         # the order on a different request).
-        order_number = self.generate_order_number(basket)
+        order_number, basket = self.generate_order_number(basket)
```

Comment:
Can you double check that `generate_order_number()` is not used anywhere else in the project, otherwise it will break due to new return type.

Reply:
It is not used anywhere else in FS or Oscar.